doc/api/tls.md

```diff
@@ -1094,6 +1094,37 @@ See
 [SSL_get_shared_sigalgs](https://www.openssl.org/docs/man1.1.1/man3/SSL_get_shared_sigalgs.html)
 for more information.
 
+### `tlsSocket.exportKeyingMaterial(length, label)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `length` {number}
+  number of bytes to retrieve from keying material
+* `label` {string} an application specific label, typically this will be a
+value from the IANA Exporter Label Registry
+(https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#exporter-labels).
+
+* Returns: {Buffer} {number} of bytes of the keying material
+
+Example
+
+```js
+const keyingMaterial = tlsSocket.exportKeyingMaterial(
+  128,
+  'ttls keying material');
+
+/**
+ Example return value of keyingMaterial:
+ <Buffer 76 26 af 99 c5 56 8e 42 09 91 ef 9f 93 cb ad 6c 7b 65 f8 53 f1 d8 d9
+    12 5a 33 b8 b5 25 df 7b 37 9f e0 e2 4f b8 67 83 a3 2f cd 5d 41 42 4c 91
+    74 ef 2c ... 78 more bytes>
+*/
+```
+See
+[SSL_export_keying_material](https://www.openssl.org/docs/man1.1.1/man3/SSL_export_keying_material.html)
```

Comment:
Can you add the link at the bottom of tls.md and then refer to it with `[SSL_export_keying_material][]` syntax?

Reply:
done